### PR TITLE
Fix hero sizes

### PR DESCRIPTION
**Pull request by victoriadotdev**
- Use proper sections in index.html to comply with bulma styles
- Update default value of header height to "fullheight" (shows navbar)

### CODE REVIEW

**Comment by er-vin** [comment]
For the record I just tested this patch locally, and that fixes the long standing issue I had with the hero sizes since the big theme refactoring. Looking forward to it getting merged!

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-exampleSite/public
+exampleSite/public/
+exampleSite/resources/
+docs/css/main*
```
Comment (hanzei) [inline .gitignore]: Why would you exclude this?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
-exampleSite/public
+exampleSite/public/
+exampleSite/resources/
```
Comment (hanzei) [inline .gitignore]: Bep suggested to commit these files into. We can discuss this, but this is out of scope if this PR for me.
